setup.py

```diff
@@ -6,7 +6,7 @@
     LONG_DESCRIPTION = f.read()
 
 setup(name='pipelinewise',
-      version='0.38.0',
+      version='0.38.1',
```

Comment:
3 changes are either breaking changes or new features, we need to bump at least the minor not the patch ... I know we can move to v1 yet so minor it is then 🤷🏽